contribute.md

```diff
@@ -121,8 +121,22 @@ Helping out other users in the "Forums" is always useful. Frequent problems or q
 
 Testing is also *very* welcome, for any issue encountered, please open a bug / feature request.
 
+### Bug reporting
+Even the simplest of bugs reported helps us make the project better. The issue tracker is located at [http://projects.theforeman.org/projects/foreman/issues](http://projects.theforeman.org/projects/foreman/issues), and you should follow these guidelines:
```

Comment:
This links to the Foreman project specifically, so I'd add a bullet point below to ensure that the correct project is chosen first.